massgen/plan_storage.py

```
@@ -167,6 +184,27 @@ def get_latest_plan(self) -> Optional[PlanSession]:
+        """Get the most recent resumable plan session, if any."""
+        if not PLANS_DIR.exists():
+            return None
+
+        for plan_dir in sorted(PLANS_DIR.glob("plan_*"), reverse=True):
+            plan_id = plan_dir.name.replace("plan_", "")
+            session = PlanSession(plan_id)
+            if not session.metadata_file.exists():
+                continue
+            try:
+                metadata = session.load_metadata()
+            except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)